File: fix_route.py
import os

path = r'C:\Users\jlcan\Desktop\Seguimiento de plataforma de seguridad Antigravity\ssoma-platform\app\api\generate-acta-scsst\route.ts'
with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace(
    'const chartUrl = https://quickchart.io/chart?c= + encodeURIComponent(JSON.stringify(chartConfig)) + &w=600&h=300&bkg=white;',
    'const chartUrl = https://quickchart.io/chart?c= + encodeURIComponent(JSON.stringify(chartConfig)) + &w=600&h=300&bkg=white;'
)

with open(path, 'w', encoding='utf-8') as f:
    f.write(content)

File: fix_route.js
const fs = require('fs');
let c = fs.readFileSync('app/api/export-excel/route.ts', 'utf8');
c = c.replace(/\\n/g, '\n');
c = c.replace(/}\s*}\s*else if\s*\(isInternas\)\s*{/, '} else if (isInternas) {');
fs.writeFileSync('app/api/export-excel/route.ts', c);
